mailbox/grpc_noise_conn.go

```diff
@@ -167,53 +166,33 @@ func (c *NoiseGrpcConn) ClientHandshake(_ context.Context, _ string,
 	// First, initialize a new noise machine with our static long term, and
 	// password.
 	var err error
-	c.noise, err = NewBrontideMachine(true, c.localKey, c.password)
+	c.noise, err = NewBrontideMachine(
+		true, XXPattern, c.localKey, c.remoteKey, c.password,
+		c.authData, HandshakeVersion,
```

Comment:
Set `c.authData` to `nil` explicitly on the client side? Since it never sends auth data.

Reply:
Also, the remote key cannot be known here yet, so we should also explicitly set to nil for clarity.